video/out/hwdec/hwdec_vaapi_gl.c

```diff
@@ -135,11 +144,18 @@ static void vaapi_gl_mapper_uninit(const struct ra_hwdec_mapper *mapper)
                         p_mapper->desc.layers[n].offset[plane]); \
             ADD_ATTRIB(EGL_DMA_BUF_PLANE ## plane ## _PITCH_EXT, \
                         p_mapper->desc.layers[n].pitch[plane]); \
+            if (p_owner->use_modifiers) {                              \
```

Comment:
Might as well also check that the modifier is set (i.e. not `DRM_MOD_INVALID`), just to be on the safe side. (I'm not sure where these modifiers come from but I remember running into a case similar like this when implementing this in my own code)